client/index.html

```diff
@@ -214,12 +214,26 @@ <h1 class="title">Settings</h1>
 							<div class="col-sm-12">
 								<h2>Visual Aids</h2>
```

Comment:
I would prefer a new "Themes" category rather than adding this to "Visual Aids".